### PR TITLE
feat: add meta description and Open Graph image (fix #146)

### LINKED ISSUE #146: Add social sharing image and meta description
## Is your feature request related to a problem? Please describe.

For SEO/sharing, we need a 1200&times;630 pixel image and a meta description for the site.

## Describe the solution you'd like

Add meta tags and an image.

## Describe alternatives you've considered

N/A.

## Additional context

N/A.

### PR DESCRIPTION
**Pull request by greatislander**
* [x] This pull request has been linted by running `npm run lint` without errors
* [x] This pull request has been tested by running `npm run dev` and reviewing affected routes
* [x] This pull request has been built by running `npm run generate` without errors
* [x] This isn't a duplicate of an existing pull request

## Description

Adds a meta description and Open Graph image. Also, properly populates data for the Web Application Manifest ([docs](https://developer.mozilla.org/en-US/docs/Web/Manifest)) with configured information.

## Steps to test

1. Inspect page head.
2. Find `og:image` meta tag.
3. Find `description` meta tag.
4. Find web application manifest.
5. Find favicon and app icons.

## Additional information

Metadata can be expanded in future but this is a good starting point. Some more docs:

- https://nuxtjs.org/api/pages-head/
- https://pwa.nuxtjs.org/modules/meta/

@lliskovoi I don't think you sent me an Open Graph image so I made a [temporary one](https://deploy-preview-155--wecount.netlify.com/og-image.png). I can swap in a replacement whenever you have a chance to make it.

## Related issues

Resolves #146.

### CODE REVIEW

File: assets/config.js [hunk 1]
```diff
@@ -4,36 +4,34 @@
...
 	socialMedias: [{
 		id: "facebook",
 		title: "Facebook",
 		href: "",
-		svg: "<svg width='78' height='78' viewBox='-2 -2 82 82' fill='none' xmlns='http://www.w3.org/2000/svg'><path d='M78 39C78 17.4586 60.5414 0 39 0C17.4586 0 0 17.4586 0 39C0 58.4695 14.2594 74.6027 32.9062 77.5277V50.2734H23.0039V39H32.9062V30.4078C32.9062 20.635 38.7258 15.2344 47.6379 15.2344C51.9035 15.2344 56.3672 15.9961 56.3672 15.9961V25.5938H51.4465C46.602 25.5938 45.0938 28.6025 45.0938 31.6875V39H55.9102L54.1811 50.2734H45.0938V77.5277C63.7406 74.6027 78 58.4695 78 39Z' fill='white'/></svg>"
+		svg: "<svg width='78' height='78' viewBox='-2 -2 82 82' xmlns='http://www.w3.org/2000/svg'><path d='M78 39C78 17.4586 60.5414 0 39 0C17.4586 0 0 17.4586 0 39C0 58.4695 14.2594 74.6027 32.9062 77.5277V50.2734H23.0039V39H32.9062V30.4078C32.9062 20.635 38.7258 15.2344 47.6379 15.2344C51.9035 15.2344 56.3672 15.9961 56.3672 15.9961V25.5938H51.4465C46.602 25.5938 45.0938 28.6025 45.0938 31.6875V39H55.9102L54.1811 50.2734H45.0938V77.5277C63.7406 74.6027 78 58.4695 78 39Z' fill='white'/></svg>"
```
Comment (jobara) [inline assets/config.js:19]: Just double checking that the intention here is to only remove `fill='none'` from `<svg>` and to leave the `fill='white'` on the `<path>`.

File: assets/config.js [hunk 2]
```diff
@@ -4,36 +4,34 @@
...
 export default {
 	appTitle: "We Count",
-	appTitleShort: "WC",
-	appDescription: "We Count Project",
-	appThemeColor: "#fffffff",
-	appBgColor: "#00172c ",
-	appIcon: "assets/app-icon.png",
+	appShortTitle: "We Count",
+	appDescription: "Removing bias and exclusion in the data economy.",
+	appThemeColor: "#fffffF",
```
Comment (jobara) [inline assets/config.js:9]:
```suggestion
	appThemeColor: "#ffffff",
```

File: nuxt.config.js [hunk 3]
```diff
@@ -85,6 +85,13 @@ export default {
 			families: ["Fira+Sans:400,400i,500,600,700&display=swap"]
 		}
 	},
+	pwa: {
+		manifest: {
+			name: config.appTitle,
+			short_name: config.appShortTitle,
+			description: config.appDescription
+		}
+	},
```
Comment (jobara) [inline nuxt.config.js:94]: Should there be manifest entries for [`icons`](https://developer.mozilla.org/en-US/docs/Web/Manifest/icons) and [`lang`](https://developer.mozilla.org/en-US/docs/Web/Manifest/lang). Are these populated by another means?

`icons` is a mandatory field. I see that we have static/icon.png but I don't see it used anywhere. Was it meant to be added for this?

`lang` isn't mandatory. It looks like it specifies the language of the text used for things like the name, short_name and description. I'm not sure how necessary this is for the PWA though.

File: nuxt.config.js [hunk 4]
```diff
@@ -40,10 +40,10 @@ export default {
 		meta: [
 			{ charset: "utf-8" },
 			{ name: "viewport", content: "width=device-width, initial-scale=1" },
-			{ hid: "description", name: "description", content: process.env.npm_package_description || "" }
+			{ hid: "og:image", property: "og:image", content: "/og-image.png" } // TODO: Customize per page.
```
Comment (jobara) [inline nuxt.config.js:43]: It seems the `og:url` meta data is missing. According to [The Open Graph Protocol](https://ogp.me) this is a required property.

File: assets/config.js (hunk 1 above)
Comment (greatislander) [reply assets/config.js]: Oops, missed that. Good catch. Fixed in a54afed.

File: nuxt.config.js (hunk 3 above)
Comment (greatislander) [reply nuxt.config.js:94]: The `icons` are generated automatically if `static/icon.png` is in place (see https://pwa.nuxtjs.org/modules/icon.html).

`lang` defaults to `en` which is fine.

File: pages/_slug/index.vue [hunk 5]
```diff
@@ -18,7 +18,11 @@ export default {
 	},
 	head () {
 		return {
-			titleTemplate: this.title + " | %s"
+			titleTemplate: this.title + " | %s",
+			meta: [
+				{ hid: "og:title", name: "og:title", content: this.title + " | We Count" },
+				{ hid: "og:url", name: "og:url", content: Config.appBaseUrl + this.$nuxt.$route.fullPath }
```
Comment (jobara) [inline pages/_slug/index.vue:24]: IT seems the `og:url` is not being assigned to the `property` attribute.  The same for `og:title` and `og:image`. See https://ogp.me/

Comment (greatislander) [reply pages/_slug/index.vue]: Resolved in 848dced.

**Comment by TeddTech** [comment]
Besides the merge conflict, this PR looks good.

**Pull request merged**